docs/RELEASING.md

```diff
@@ -0,0 +1,77 @@
+# How to make a Release
+
+- [How to make a Release](#how-to-make-a-release)
+  - [Updating](#updating)
+    - [Updating Java Version](#updating-java-version)
+  - [Versioning](#versioning)
+  - [Changelog](#changelog)
+    - [Style](#style)
+    - [Template](#template)
+  - [Releasing Steps](#releasing-steps)
+
+
+## Updating
+
+New features and bugfixes PRs should always be made against `main` branch.
+
+### Updating Java Version
+
+When updating the java version don't forget to update also `jitpack.yml` to make use of the correct jdk because jitpack sometimes has a hard time defining which image it should use
+
+## Versioning
+
+Version should always be changed in `gradle.properties` file, following the [Semantic Versioning 2.0.0](https://semver.org/#semantic-versioning-200), in short:
+
+Version format - MAJOR.MINOR.PATCH
+
+ - MAJOR: Incompatible API Changes
+ - MINOR: Backward compatible new functionality
+ - PATCH: Backward compatible bug fixes
+
+## Changelog
+
+The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/). All unreleased changes should be in `NEXT` section with the same pattern.
+
+### Style
+
+For consistency, all CHANGELOG entries should follow a common style:
+
+- Use `##` for the version line. A version line should have a blank line before and after it.
+- Use `-` for individual items.
+- Entries should end with a `.`.
+- Breaking changes should be introduced with **BREAKING CHANGE**:, or **BREAKING CHANGES**: if there is a sub-list of changes.
+
+
+### Template
+
+```markdown
+
+## [NEXT]
+
+ - Upcoming feature already merged (#125).
+
+## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
+
+### Added/Changed/Deprecated/Fixed/Removed/Security
+
+ - Adds some feature (#123).
+ - **BREAKING CHANGE**:
+   - Removes older content (#124).
+
+...
+
+[MAJOR.MINOR.PATCH]: https://github.com/profusion/android-enhanced-video-player/releases/tag/[MAJOR.MINOR.PATCH]
+
+```
+
+
+## Releasing Steps
+
+ 1. Create a new branch (e.g. release/0.1.0)
+ 2. Update `VERSION_NAME` in `gradle.properties`
+ 3. Update `CHANGELOG.md` renaming the `Unreleased` section to the new version, following the [Changelog Template](#changelog-template) pattern
+ 4. Add new release link to the bottom of `CHANGELOG.md` file
+ 5. Push the changes, create a PR and merge it to `main` branch
+ 6. Add a tag with the new release, either locally or via [GitHub](https://github.com/profusion/android-enhanced-video-player/releases/new)
+ 7. Open [JitPack](https://jitpack.io/#profusion/android-enhanced-video-player) and check if the current version status is OK
+ 8. Create issues in project that uses our library notifing the new version released
```

Comment:
Leave this to the hands of whoever is managing their own project, on this one we only care about publishing a new version.